css/_typo.scss

```diff
@@ -48,6 +48,7 @@
 //
 //To support all manner of languages we use <a href="https://github.com/angular-translate/angular-translate">angular-translate</a>.
 //It allows us to translate in templates using {&nbsp;{'Key'|translate}&nbsp;} or $translate.instant('Key') when translating from JavaScript.
+//Keys only used in this styleguide should be referenced in the styleguide_translation_keys.html file in the project root.
```

Comment:
I think it should be `Keys used in this styleguide` instead of ~~`Keys only used in this styleguide`~~ because keys can be used in both project and styleguide. When we deleting some generic keys like 'From' or 'Start' from project (commonly it is a rare case), we can find out that they exist in styleguide_translation_keys.html which means they are used in styleguide as well, so we can not delete them. Then these keys are not only used in styleguide, they are used by both.